shareable-dataframes/Dockerfile

```diff
@@ -60,11 +58,13 @@ RUN conda env create -f /rapids/conda/shareable_dataframes.yml --name shareable_
 ADD . /rapids/
 
 # Configure
+## 75->Tuning
+## check if NATIVE should fix it
 RUN source activate shareable_dataframes \
     && cmake \
     -DBUILD_TESTS=OFF \
     -DBUILD_BENCHMARKS=OFF \
-    -DCMAKE_CUDA_ARCHITECTURES= \
+    -DCMAKE_CUDA_ARCHITECTURES=${GPU_ARCH} \
```

Comment:
Automatic detection with empty DCMAKE_CUDA_ARCHITECTURES was failing. In my tests, I get
Failed to find a working CUDA architecture. Manually fixing seems to do the trick.

@randerzander, Would it be fine if we make this an argument that the user provides/configures on top of the docker file ?

Reply:
Resolved offline.